ui/webui/src/components/Error.jsx

```diff
@@ -36,6 +36,8 @@ import { ExternalLinkAltIcon } from "@patternfly/react-icons";
 
 import { exitGui } from "../helpers/exit.js";
 
+import "./Error.scss";
```

Comment:
Would it also work by using the Patternfly pre solution?    ` <Text component={TextVariants.pre}> `

Reply:
I tried this:
```
                    <TextContent id="critical-error-review-details">
                        <Text component={TextVariants.pre}>
                            {exception.name + ": " + exception.message}
                        </Text>
                    </TextContent>
```
![Screenshot from 2023-08-11 09-09-43](https://github.com/rhinstaller/anaconda/assets/1220237/54624457-40a7-46e7-a100-c0dde7a886a8)

Reply:
oops does not look good. Sorry for wrong hint